cmd/main.go

```diff
@@ -70,6 +71,10 @@ func setupFlags(name string) (*pflag.FlagSet, *options) {
 		"use different icons, see help for options")
 	flags.BoolVar(&opts.rawCommand, "raw-command", false,
 		"don't prepend 'go test -json' to the 'go test' command")
+	flags.BoolVar(&opts.readStdin, "stdin", false,
+		"don't run any command, instead read go test stdout from stdin")
+	flags.IntVar(&opts.readStderrFD, "stderr", 0,
```

Comment:
Should this be a named file instead?

Perhaps some elaborate invocation could work without a named pipe. In practice, I think with bash one has to rely on one.